monkeys/alexa-top-dns/templates/fqdn-policy.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: chaos-test-{{ .Values.basename }}
+spec:
+  endpointSelector:
+    matchLabels:
+      name: {{ .Values.name }}
+  egress:
+  - toPorts:
+      - ports:
+         - port: "53"
+           protocol: ANY
+        rules:
+          dns:
+            - matchPattern: "*.*"
+            - matchPattern: "*.*.*"
```

Comment:
The latest nightlies now have support for `matchPattern: "*"`. These are actually just fine, but if you wanted it even simpler we could use that.